### DISCUSSION TITLE
Peppol SMP 1.3.0 implementation

### DISCUSSION THREAD

**Discussion by skizub** [phax/phoss-smp#285] [General]
Hello,

Based on [PEPPOL 1.3.0 Change Impact Document](https://docs.peppol.eu/edelivery/changelog/2023-11/SMP%201.3.0%20Impact%20Analysis%20v1.1.pdf) the change is about replacing SHA1 with SHA256 for signature and digest methods. Deadline for SMP protocol update was defined as June 2024.

Does latest version of Phoss SMP server (7.1.3) support PEPPOL SMP 1.3.0?

If yes,
1. Will it work out of the box after server update or is there a need to make changes to server configuration additionally?
2. What is an easiest way to check if server is already using new specifications?

Many thanks in advance.

**Comment by phax** [comment]
@skizub indeed, phoss SMP 7.1.x supports the Peppol 1.3.0 specification.
See https://github.com/phax/phoss-smp/wiki/News-and-noteworthy for the 7.1.0 release.

Regarding potential issues on update, please see https://github.com/phax/phoss-smp/wiki/Migrations

**Comment by skizub** [comment]
@phax  - thank you.
